tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -72,6 +73,9 @@ def close_app(self, app):
         self.wait_for_card_ready(app)
         self.marionette.find_element(*self._close_button_locator(app)).tap()
         self.wait_for_element_not_present(*self._app_card_locator(app))
+        if len(self.cards) == 0:
+            self.marionette.switch_to_frame()
+            self.wait_for_condition(lambda m: self.apps.displayed_app.name == Homescreen.name)
```

Comment:
I think this condition should be added in the test, along with the import, as in [test_sms_auto_save_draft.py][link]
[link]:https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_auto_save_draft.py#L46